src/main/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializer.java

```diff
@@ -29,17 +30,16 @@ public class RandomTxPowerInitializer extends TPC {
 	/** The RRM algorithm ID. */
 	public static final String ALGORITHM_ID = "random";
 
-	/** Default tx power. */
-	public static final int DEFAULT_TX_POWER = 23;
-
 	/** The fixed tx power (dBm). */
 	private final int txPower;
 
 	/** Constructor (uses default tx power). */
 	public RandomTxPowerInitializer(
 		DataModel model, String zone, DeviceDataManager deviceDataManager
 	) {
-		this(model, zone, deviceDataManager, DEFAULT_TX_POWER);
+		super(model, zone, deviceDataManager);
+		Random rand = new Random();
+		this.txPower = rand.nextInt(TPC.MAX_TX_POWER + 1 - TPC.MIN_TX_POWER) + TPC.MIN_TX_POWER;
```

Comment:
nit: I would write a sub-method `int getRandomTxPower()` to extract that part out, then this constructor can be simplified to 1 line:
```
this(model, zone, deviceDataManager, getRandomTxPower());
```